src/Client/AzureManaged/DurableTaskSchedulerClientExtensions.cs

```diff
@@ -85,6 +85,12 @@ static void ConfigureSchedulerOptions(
             .Configure(additionalConfig ?? (_ => { }))
             .ValidateDataAnnotations();
 
+        builder.Services.AddOptions<DurableTaskClientOptions>(builder.Name)
```

Comment:
I think there is a `builder.Configure(optionsCallback)` overload you can use

Reply:
yes, much better.